src/external/views/nunjucks/returns/return.njk

```diff
@@ -0,0 +1,57 @@
+{% extends "./nunjucks/layout.njk" %}
+{% from "returns/not-current-version-warning.njk" import notCurrentVersionWarning %}
+{% from "returns/meter-details-inset-text.njk" import meterDetailsInsetText %}
+{% from "returns/return-header.njk" import returnHeader %}
+{% from "returns/return-quantities-table.njk" import returnQuantitiesTable %}
+{% from "details/macro.njk" import govukDetails %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+      {{ notCurrentVersionWarning(data) }}
+      {{ returnHeader(data, documentHeader, showMeta) }}
+    </div>
+
+    <div class="govuk-grid-column-full">
+      {% if isVoid %}
+        <h2 class="govuk-heading-l">This return has been made void</h2>
+      </div>
+    {% else %}
+      {% if data.isNil %}
+        <h2 class="govuk-heading-l">Nil return</h2>
+      {% else %}
+        <h2 class="govuk-heading-m">Abstraction volumes</h2>
+      {% endif %}
+    </div>
+
+    <div class="govuk-grid-column-full">
```

Comment:
I think there is a standard contact details macro somewhere we could use with email/phone etc.